kvmagent/kvmagent/plugins/tensorfusion/container_executor.py

```
@@ -0,0 +1,514 @@
+            sm_percent_limit=request.sm_percent_limit,
+            shm_size_mb=shm_size_mb,
+            log_file=log_file,
+            enable_log=enable_log,
+            log_level=log_level,
+            license_value=request.license,
+            license_sign=request.license_sign,
+        )
+
+        try:
+            container_id = self._docker(cmd).strip()
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)